### PR TITLE
HwF - Demo image policy update

### PR DESCRIPTION
**Pull request by zaparka**

### CODE REVIEW

File: apps/help-with-fees/help-with-fees-staffapp/demo.yaml [hunk 1]
```diff
@@ -7,3 +7,4 @@ spec:
     base:
       environment:
         Trig_1: Demo1
+      image: hmctspublic.azurecr.io/help-with-fees/staffapp:feature/UCD-work-mvp #{"$imagepolicy": "flux-system:demo-help-with-fees-staffapp"}
```
Comment (timja) [inline apps/help-with-fees/help-with-fees-staffapp/demo.yaml:10]: this won't work the image will be a pr-ddd image:
https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md#deploy-non-prod-image-to-an-environment

File: apps/help-with-fees/help-with-fees-publicapp/demo-image-policy.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+apiVersion: image.toolkit.fluxcd.io/v1beta1
+kind: ImagePolicy
+metadata:
+  name: demo-help-with-fees-publicapp
+spec:
+  imageRepositoryRef:
+    name: help-with-fees-publicapp
+  filterTags:
+    pattern: '^pr-678-[a-f0-9]+-(?P<ts>[0-9]+'
```
Comment (zaparka) [inline apps/help-with-fees/help-with-fees-publicapp/demo-image-policy.yaml:9]: @timja I added this section, do I need it or is the image in demo.yml enough? also the "tag" is created by me in GitHub or by something else?

Comment (timja) [reply apps/help-with-fees/help-with-fees-publicapp/demo-image-policy.yaml:9]: tag is created by your pull request on the jenkins build

**Comment by zaparka** [comment]
@timja I accepted your comments. PR is passing now

**Review by timja — APPROVED** [review]

**Review by cakeben — APPROVED** [review]

**Pull request merged**